web-animations-1/Overview.bs

```diff
@@ -3236,7 +3242,7 @@ Before calculating the <a>effect value</a> of a <a>keyframe effect</a>,
 the property values specified on its <a>keyframes</a> are resolved to
 <a>computed values</a>, and the offset to use for any keyframes with a null
 <a>keyframe offset</a> is computed. The result of resolving these values is
-a set of <dfn>computed keyframes</dfn>.
+a set of <dfn>computed keyframes</dfn> for a particular moment in time.
```

Comment:
Not sure this addition is needed?

Reply:
I was trying to clarify something that got misread and lead to bugs in our current implementation.

Reply:
Can you elaborate on that? I suspect there's probably a better way to clarify in that case.

Reply:
I think there is, I plan on proposing an (isomorphic) refactor to this section to clarify it as a separate patch. Perhaps this should wait until then. The idea was that this depends on what the inherited styles are at when they're calculated, which must be the same time they're used (i.e. a keyframes do not capture parent styles at the moment of their creation).

Reply:
Ok that makes sense. And it probably would be best as a separate refactor.